doc/releases.rst

```diff
@@ -9,6 +9,41 @@
 Releases
 ========
 
+Version 0.14.4
+==============
+
+* Fix #1356: SVR degree hyperparameter now only active with "poly" kernel.
+* Add #1311: Black format checking (non-strict).
+* Maint #1306: Run history is now saved every iteration incase of crashes.
+* Doc #1309: Updated doc faqs and manual
```

Comment:
Could you please be a bit more concrete here?

Reply:
Sure